code42.json

```diff
@@ -1924,6 +1924,463 @@
+                    "data_type": "string",
+                    "example_values": [
+                        "2021-01-01T12:00:00.000000Z"
+                    ],
+                    "column_order": 5
+                },
+                {
+                    "column_name": "Created By",
+                    "data_path": "action_result.data.*.createdByUsername",
+                    "data_type": "string",
+                    "example_values": [
+                        "admin@example.com"
+                    ],
+                    "contains": ["email"],
+                    "column_order": 6
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary.total_count",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "add case event",
+            "identifier": "add_case_event",
+            "description": "Associates a file event with a Code42 case.",
+            "type": "investigate",
+            "read_only": false,
+            "parameters": {
+                "case_number": {
+                    "description": "Case number",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "case number"
+                    ],
+                    "order": 0
+                },
+                "event_id": {
+                    "description": "Event ID",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": ["file event"],
+                    "order": 1
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.case_number",
+                    "data_type": "string",
+                    "column_name": "Case Number",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.parameter.event_id",
+                    "data_type": "string",
+                    "column_name": "Event ID",
+                    "column_order": 1
+                },
+
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
```

Comment:
I think the summary fields should be listed out, like `summary.case_number` etc.
But we are repeating a lot of info in the summary that is already visible in the table and usable in the playbooks and stuff, might be worth just sticking with a case number for a summary and including that in this JSON as such